tests/src/strategy.spec.ts

```diff
@@ -3,16 +3,61 @@
  * with very little code change. Most of the test cases
  * are already covered.
  */
-import { testData, setupExpressApp } from '../helpers/setupApp';
+import { setupExpressApp } from '../helpers/setupApp';
 import supertest from 'supertest';
 import passport from 'passport';
+import SnykOAuth2Strategy from '../../src';
+
+/**
+ * Data required fro Snyk OAuth2 Strategy.
+ * It is used in other places, so stored in
+ * a separate constant.
+ */
+export const testData = {
+  authorizationURL: 'https://example_authorization.com/authorize',
+  tokenURL: 'https://example_token.com/token',
+  clientID: 'test',
+  clientSecret: 'test',
+  callbackURL: 'https://example_callback.com',
+  scope: 'apps:beta',
+  nonce: 'some_nonce_value',
+};
 
 describe('Strategy with passport', () => {
-  const app = setupExpressApp();
+  const snykOAuth2 = new SnykOAuth2Strategy(
+    {
+      authorizationURL: testData.authorizationURL,
+      tokenURL: testData.tokenURL,
+      clientID: testData.clientID,
+      clientSecret: testData.clientSecret,
+      callbackURL: testData.callbackURL,
+      scope: testData.scope,
+      scopeSeparator: ' ',
+      state: true,
+      passReqToCallback: true,
+      nonce: testData.nonce,
```

Comment:
should the nonce value still be here as it's required per-request and different for each per the spec?

Reply:
POC 🙂 will remove once this method is feasible for all